project/npda/general_functions/csv_upload.py

```diff
@@ -782,311 +782,293 @@ def validate_date(
+                ]
+                if not pd.isnull(
                     row[
                         "If treatment included insulin pump therapy (i.e. option 3 or 6 selected), was this part of a closed loop system?"
                     ]
-                    if not pd.isnull(
-                        row[
-                            "If treatment included insulin pump therapy (i.e. option 3 or 6 selected), was this part of a closed loop system?"
-                        ]
-                    )
-                    else None
-                ),
-                "glucose_monitoring": row[
-                    "At the time of HbA1c measurement, in addition to standard blood glucose monitoring (SBGM), was the patient using any other method of glucose monitoring?"
-                ],
-                "systolic_blood_pressure": row["Systolic Blood Pressure"],
-                "diastolic_blood_pressure": row["Diastolic Blood pressure"],
-                "blood_pressure_observation_date": (
-                    row["Observation Date (Blood Pressure)"]
-                    if not pd.isnull(row["Observation Date (Blood Pressure)"])
-                    else None
-                ),
-                "foot_examination_observation_date": (
-                    row["Foot Assessment / Examination Date"]
-                    if not pd.isnull(row["Foot Assessment / Examination Date"])
-                    else None
-                ),
-                "retinal_screening_observation_date": (
-                    row["Retinal Screening date"]
-                    if not pd.isnull(row["Retinal Screening date"])
-                    else None
-                ),
-                "retinal_screening_result": row["Retinal Screening Result"],
-                "albumin_creatinine_ratio": row["Urinary Albumin Level (ACR)"],
-                "albumin_creatinine_ratio_date": (
-                    row["Observation Date: Urinary Albumin Level"]
-                    if not pd.isnull(row["Observation Date: Urinary Albumin Level"])
-                    else None
-                ),
-                "albuminuria_stage": row["Albuminuria Stage"],
-                "total_cholesterol": row["Total Cholesterol Level (mmol/l)"],
-                "total_cholesterol_date": (
-                    row["Observation Date: Total Cholesterol Level"]
-                    if not pd.isnull(row["Observation Date: Total Cholesterol Level"])
-                    else None
-                ),
-                "thyroid_function_date": (
-                    row["Observation Date: Thyroid Function"]
-                    if not pd.isnull(row["Observation Date: Thyroid Function"])
-                    else None
-                ),
-                "thyroid_treatment_status": (
+                )
+                else None
+            ),
+            "glucose_monitoring": row[
```

Comment:
This code is the same as before, just at one level of indentation lower. It was tempting to do some additional refactoring here but I'm leaving it for #31